pep/app/pods/most-cited-loading/template.hbs

```diff
@@ -0,0 +1,3 @@
+<Page::Content>
+    <Loading::Read::Document />
```

Comment:
Note for future work, but we should make a `<Loading::Table />` component to use for these data table pages. We have a custom skeleton placeholder component for tables that i made extending from the addon, and have it in use in CC/J&R/etc